cpg_workflows/jobs/genotype.py

```diff
@@ -189,6 +189,15 @@ def _haplotype_caller_one(
     # of a Hail worker (2 cores), plus with the `highmem` instance that would
     # give enough memory: 13G. That's not going to give enough disk storage, so we
     # are explicitly requesting more storage.
+    #
+    # Based on an audit of RD crams on 19/05/23, 99% of crams are <34Gb. Will set the
+    # default to 40Gb for genomes then use a run specific confg to run the rare
+    # sample that will fail from this limit.
```

Comment:
```suggestion
    # sequencing group that will fail from this limit.
```
In anticipation for this change coming v. soon https://github.com/populationgenomics/production-pipelines/pull/366